cruise.umple/src/generators/Generator_CodePlainRequirementsDoc.ump

```diff
@@ -27,7 +30,30 @@ class PlainRequirementsDocGenerator
   void generate(){
     doc.createParagraphElement("Plain Requirements Doc from "+model.getUmpleFile().getSimpleFileName()+".ump", false);
     UmpleModelWalker walker = new UmpleModelWalker(model);
-    walker.accept(new ReqVisitor(doc));
+    //System.out.println(model.getAllRequirements().values());
+    TreeSet<Requirement> ReqTreeSet;
+    if (model.modelHasSuboption("PlainRequirementsDoc", "reqSortID")) {
+      Comparator<Requirement> IdComparator = (r1, r2) -> r1.getIdentifier().compareTo(r2.getIdentifier());
+      // Initialize with ID Comparator
+      ReqTreeSet = new TreeSet<>(IdComparator); 
```

Comment:
Tis statement is the same as statement 43 and 48 below so should be done after the if/else blocks